src/handlers/comment/handlers/ask.ts

```diff
@@ -0,0 +1,74 @@
+import OpenAI from "openai";
+import { getBotContext, getLogger } from "../../../bindings";
+import { Payload } from "../../../types";
+
+const openai = new OpenAI({
+  apiKey: process.env.OPENAI_API_KEY,
+});
+
+const sysMsg = `You are the Ubiquity AI, designed to provide accurate technical answers. \n
```

Comment:
Pretty decent prompt but I'm concerned with the amount of times you clarified that it should assume it has enough context to answer the question. Perhaps we should be focused on giving it more context, like pasting in the entire conversation history from the pull request review or original issue.